### ISSUE TITLE
[Windows and Linux] libcrypto-3-x64 missing

### ISSUE THREAD

**Issue by FMotalleb** [isar/isar#1352]
### Environment

 - Platform: Windows (tested on win-11-x64)
 - Flutter version: 3.10.5
 - Isar version: [4.0.0-dev.1](https://pub.dev/packages/isar_flutter_libs/versions/4.0.0-dev.1)

---

There is a dependency issue in the `isar.dll` file:
```
[ROOT] isar.dll : isar.dll
...
[WindowsFolder] VCRUNTIME140.dll : C:\WINDOWS\system32\VCRUNTIME140.dll
[NOT_FOUND] libcrypto-3-x64.dll :
```
As mentioned above, the `libcrypto-3-x64.dll` file is not included in the standard Windows distribution. This means that users must download it separately and place it in the same directory as the `isar.dll` file. This can be inconvenient and messy, especially if the user is not familiar with how to install DLL files.

### HOTFIX:
If you are facing the same issue as mentioned, you can download the `libcrypto-3-x64.dll` file from a website like [DLL-files.com](https://www.dll-files.com/libcrypto-3.dll.html) or [download](https://slproweb.com/products/Win32OpenSSL.html) Win32OpenSSL. Once you have downloaded the file, you need to place it inside your output directory, next to the `isar.dll` file.

### Reproduce

1. Make a backup of the `libcrypto-3-x64.dll` file.
  Locate the file in one of the following locations:
  `C:\windows\System32\libcrypto-3.dll`
  `C:\Windows\SysWOW64\libcrypto-3-x64.dll`
2. Delete the file.
3. Try to run an application.

**Comment by angelocordero** [comment]
I was the one who reported this error on the telegram group. If this hasn't been fixed already, I suggest changing the title into something that can easily be seen when searching the error, `IsarNotReadyError: Could not initialize IsarCore library for processor architecture "windows x64"`, making it easier to be seen by people who are unaware that the issue is because of the missing DLL

**Comment by simc** [comment]
This is somewhat fixed in 4.0.0-dev.5. Encryption is temporarily disabled for Windows until I find a proper fix

**Comment by abhay-s-rawat** [comment]
I think leaving the error will be better than disabling it because of it developer will know that issue is there and can hold the release of the app. What do you think simc ? Can you please tell me which encryption is used , is it AES-256 ?

**Comment by FMotalleb** [comment]
@abhay-s-rawat Devs shouldn't use unstable packages in their prods.

**Comment by simc** [comment]
This is fixed in 4.0.0-dev.11
